### PR TITLE
use Es6 for awesome books

### PR DESCRIPTION
**Pull request by george827**
### Milestone: Use ES6 modules to write modular JavaScript

- Use ES6 syntax.
- Divide your code into modules.
- Refactor your methods and functions to arrow functions.
- Use let and const in the correct way.

### CODE REVIEW

File: modules/dynamic.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            this.removeBook(bookTag);
+          });
+          const span2 = document.createElement('span');
+          span2.appendChild(input);
+          const li = document.createElement('li');
+          li.appendChild(span0);
+          li.appendChild(span1);
+          li.appendChild(span2);
+          el.appendChild(li);
+        });
```
Comment (AmaduKamara) [inline modules/dynamic.js]: - [ ] You are missing out on the functionality for adding a book to the books array and storing it to localStorage which is later fetched and rendered on the list page. Kindly work on adding a book and loading added books on the front of the list page as expected 👍

File: index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  });
+  window.addEventListener('storage', () => {
+    dmMethods.loadBooks();
+  });
+  dmMethods.loadBooks();
+
+  const addbk = document.querySelector('#add-bk');
+
+  addbk.addEventListener('click', toList);
```
Comment (AmaduKamara) [inline index.js]: - [ ] The form is not submitting data at the moment. After entering book data and hitting the submit button, the page gets refreshed and redirected me to the list page but no data is saved to localStorage and rendered on the list page. Kindly implement that in order to complete the requirement 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @george827,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Good descriptive PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.js (hunk 2 above)
Comment (george827) [reply index.js]: @AmaduKamara , The form is submitting the data. And all the books are stored in the local storage and displayed on the list. 
Screenshot (178)

Comment (AmaduKamara) [reply index.js]: Hi @george827, I tried it on my end the data is not submitted. I get redirected to the list view and no data is shown and when I get back to the form view, the form fields still retain entered data. That is how it behaves.
Did you push your last changes before you sent them for review?
I did try it and even checked the localStorage but no data persisted. Kindly recheck things again and if all is fine, add and remove works fine, then you can submit again for review 👍

**Review by prolajumokeoni — APPROVED** [review]
Hi @george827 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Branch name should have a name of a feature or action related to your project  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**